### PR TITLE
Developp

### PR DESCRIPTION
**Pull request by Abdel63**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,37 +19,39 @@
...
         @frederiquelamazone
       </a>
     </h1>
 
     <!-- CONTENU -->
     <div class="flex justify-between">
-      <div class="flex align-center justify-center ml-12">
-        <img src="images/arrow.png" class="w-24" />
+      <div
+        class="hidden sm:hidden md:flex lg:flex xl:flex align-center justify-center ml-12 pb-8 w-auto lg:w-20 xl:w-20"
```
Comment (imTHKNG) [inline index.html:31]: `hidden` and `sm:hidden` are redundant.
`md:flex`, `lg:flex` and `xl:flex` are redundant.
`lg:w-20` and `xl:w-20` are redundant.

Read this: https://tailwindcss.com/docs/responsive-design.

File: index.html [hunk 2]
```diff
@@ -58,16 +60,18 @@ <h1 class="text-center mt-2 text-xl text-gray-200 font-semibold">
...
           target="_blank"
         >
           Télécharge ton Quizz pour libérer ton Âme Amazone
         </a>
       </div>
 
-      <div class="flex align-center justify-center mr-12">
-        <img src="images/arrow.png" class="w-24" />
+      <div
+        class=" hidden sm:hidden md:flex lg:flex xl:flex align-center justify-center mr-12 pb-8 w-auto lg:w-20 xl:w-20"
```
Comment (imTHKNG) [inline index.html:72]: Same than last comment

File: main.css [hunk 3]
```diff
@@ -3,3 +3,36 @@
...
+  .rotate-45 {
+    transform: rotate(45deg);
+  }
+  .-rotate-15 {
+    transform: rotate(-15deg);
+  }
+}
+
+@responsive {
+  .text-shadow {
```
Comment (imTHKNG) [inline main.css:29]: Irrelevant CSS classname. Maybe name it with like `text-shadow-red`.

File: main.css [hunk 4]
```diff
@@ -3,3 +3,36 @@
...
+    text-shadow: 2px 2px #ff0000;
+  }
+}
+
+@responsive {
+  .linear {
```
Comment (imTHKNG) [inline main.css:35]: Irrelevant CSS classname.
Maybe name it like `linear-gradient-primary` as you used the primary colors.

File: index.html [hunk 5]
```diff
@@ -19,37 +19,39 @@
     <h1 class="text-center mt-2 text-xl text-gray-200 font-semibold">
       <a
         href="https://www.instagram.com/frederiquelamazone/"
-        class="rounded-lg bg hover:text-blue-800 hover:bg-white"
+        class="rounded-lg bg hover:text-blue-800 text-shadow"
```
Comment (imTHKNG) [inline index.html:22]: Text is blue on hover => very bad contrast.
Text-shadow in red is not readable!

image

image

**Review by KNTH01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (imTHKNG) [reply index.html:22]: I don't see your commit

File: index.html (hunk 1 above)
Comment (Abdel63) [reply index.html:31]: done

File: index.html (hunk 2 above)
Comment (Abdel63) [reply index.html:72]: done

File: main.css (hunk 4 above)
Comment (Abdel63) [reply main.css:35]: done

**Pull request merged**